lib/MusicBrainz/Server/Data/WikidataProperties.pm

```diff
@@ -0,0 +1,51 @@
+package MusicBrainz::Server::Data::WikidataProperties;
+use Moose;
+use namespace::autoclean;
+
+use Readonly;
+
+with 'MusicBrainz::Server::Data::Role::MediaWikiAPI';
+
+Readonly my $WIKIDATA_CACHE_TIMEOUT => 60 * 60 * 24 * 3; # 3 days
+
+sub get_wikidata_properties {
+    my ($self, $entity, $property) = @_;
+
+    my $url_pattern = "https://www.wikidata.org/w/api.php?action=wbgetclaims&format=json&entity=%s%s";
+    return $self->_fetch_cache_or_url($url_pattern,
+                                      "wikidata_property",
```

Comment:
This is basically a cache key. Might be a bit confusing due to how `MediaWikiAPI` is written.